lib/c/build/Dockerfile

```diff
@@ -26,16 +26,15 @@ RUN llvmParallelJobs="${llvmParallelJobs:-$(nproc)}" && \
     rm -rf /tmp/* && \
     apk --purge del build-deps
 
-COPY . $HOME/.symbolica/build
+COPY . $HOME/.symbolica
 
 # Install findutils because the default busybox xargs doesnt handle large inputs well
 # and it ends up batching the calls to llvm-link which results in us only producing
 # a libc.bc file with the contents of the final invocation in it.
-RUN apk add --no-cache --virtual musl-build-deps git make findutils && \
-    cd $HOME/.symbolica/build/install &&  \
-    ./musl.sh && \
+RUN apk add --no-cache --virtual musl-build-deps findutils git make && \
+    cd $HOME/.symbolica && \
+    ./install-musl.sh && \
     cd / && \
-    rm -rf $HOME/.symbolica/build/install && \
```

Comment:
Are we not cleaning up installation scripts anymore?

Reply:
I brought it back.